samcli/commands/pipeline/init/interactive_init_flow.py

```diff
@@ -149,11 +146,11 @@ def _read_app_pipeline_templates_manifest(pipeline_templates_dir: Path) -> Pipel
     Returns:
         The manifest of the pipeline templates
     """
-    manifest_path: Path = Path(os.path.normpath(pipeline_templates_dir.joinpath("manifest.yaml")))
-    return PipelineTemplatesManifest(manifest_path=manifest_path)
+    manifest_path: Path = Path(pipeline_templates_dir.joinpath("manifest.yaml"))
```

Comment:
`joinpath()` itself returns a Path, I don't think we need to wrap the return value in `Path(...)` again.
```suggestion
    return PipelineTemplatesManifest(pipeline_templates_dir.joinpath("manifest.yaml")
```